src/main/java/command/QuizCommand.java

```diff
@@ -0,0 +1,56 @@
+package command;
+import Dictionary.WordBank;
+import exception.WordBankNotEnoughForQuizException;
+
+import java.util.*;
+
+/**
+ * Generate a 4-option MCQ. User should answer by index of options, namely 1, 2, 3, 4.
+ */
+public class QuizCommand extends Command  {
+    public String question;
+    public String answer;
+    public String[] options;
+    protected Random random;
+    protected ArrayList<Integer> randomNumbers;
+    public int optionSequence;
+
+    public QuizCommand(){
+        this.random = new Random();
+        this.options = new String[4];
+        this.randomNumbers  = new ArrayList<>();
+    }
+
+    private boolean isDuplicate(int r){
+        for(int i=0; i<randomNumbers.size(); i++){
+            if(randomNumbers.get(i).equals(r)){
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public void generateQuiz(WordBank wordBank) throws WordBankNotEnoughForQuizException {
```

Comment:
Header comment. Also, for each rough section of code, explain what you're doing to setup the quiz